site/en/docs/privacy-sandbox/fledge/index.md

```diff
@@ -6,16 +6,20 @@ subhead: >
 description: >
   FLEDGE satisfies remarketing use cases, but is designed so it cannot be used by third parties to track user browsing behaviour across sites. The API enables on-device "auctions" by the browser, to choose relevant ads provided by websites the user has previously visited.
 date: 2021-05-18
-updated: 2021-09-10
+updated: 2021-09-16
 authors:
   - samdutton
 ---
 
-<!--lint disable no-smart-quotes-->
-
 {% Aside %}
-FLEDGE is the first experiment to be implemented in Chromium within the 
-[TURTLEDOVE](https://github.com/WICG/turtledove) family of proposals.
+FLEDGE can be tested in Chrome from versions 91.0.4472.49 or 92.0.4493.0 with the following flags enabled:
+
+`--enable-features=FledgeInterestGroups,FledgeInterestGroupAPI`
+
+For the best debugging experience, we recommend using the latest [Chrome Canary](https://www.google.com/chrome/canary).
```

Comment:
I know this is in the meeting notes, but could we either clarify what's available in Canary (is it some form of worklet debugging?) or just remove.

Reply:
Yeah... This does sound a bit odd. I think I'll delete it.